cdap-app-templates/cdap-etl/cdap-data-pipeline/src/test/java/co/cask/cdap/datapipeline/LineageOperationProcessorTest.java

```diff
@@ -18,11 +18,15 @@
 
 import co.cask.cdap.api.lineage.field.EndPoint;
 import co.cask.cdap.api.lineage.field.InputField;
+import co.cask.cdap.api.lineage.field.Operation;
```

Comment:
No logic changed in `LineageOperationProcessorTest`. Just the indentation changes and rename of the variables which fits the new plugins API. Feel free to skim through the changes in this class.